docs: Search Metadata Batch 04 (#14596)
hashicorp/consul: website/content/docs/connect/transparent-proxy.mdx

---
layout: docs

> page_title: Transparent Proxy | Service Mesh

Reviewer: ```suggestion
page_title:  Service Mesh: Enable Transparent Proxy Mode
```

description: >-

> Learn how transparent proxies enable Consul on Kubernetes to direct inbound and outbound traffic through the service mesh. Use a transparent proxy to increase application security without configuring individual services and intentions.

Reviewer 2: ```suggestion
Learn how transparent proxy enables Consul on Kubernetes to direct inbound and outbound traffic through the service mesh. Use transparent proxying to increase application security without configuring individual upstream services.
```

Still need to configure intentions.

---

> # Transparent Proxies in a Service Mesh

Reviewer 2: Transparent Proxy is a mode/feature, not a type of proxy. So this should be "Transparent Proxying in a service mesh"

Reviewer 3: ```suggestion
# Transparent Proxying in a Service Mesh
```

> # Transparent Proxies in a Service Mesh

Reviewer: ```suggestion
# Enable Transparent Proxy Mode
```
Maybe?
